lua/lspconfig/server_configurations/nim-langserver.lua

```diff
@@ -0,0 +1,23 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'nim-langserver' },
+    filetypes = { 'nim' },
+    root_dir = function(fname)
+      return util.root_pattern '*.nimble'(fname) or util.find_git_ancestor(fname)
+    end,
+    single_file_support = true,
+  },
+  docs = {
+    description = [[
+https://github.com/nim-lang/langserver 
+
+
+`nim-langserver` can be installed via the `nimble` package manager:
+```sh
+```
```

Comment:
```suggestion
nimble install nimlangserver
```

Reply:
> change file name `nim-langserver` to `nim_ls` or `nim_langserver`

Given therese a nimls.lua already, I renamed to nim_langserver